source/Octopus.Tentacle.Tests.Integration/Util/Builders/Decorators/Proxies/MethodUsageProxyDecorator.cs

```diff
@@ -37,26 +36,16 @@ public static object Create(Type scriptServiceType, object targetService, IRecor
             return proxiedService!;
         }
 
-        protected override void OnStartingInvocation(MethodInfo targetMethod)
-        {
-            usages.RecordCallStart(targetMethod);
-        }
-
-        protected override Task OnStartingInvocationAsync(MethodInfo targetMethod)
+        protected override async Task OnStartingInvocationAsync(MethodInfo targetMethod, object? request)
         {
+            await Task.CompletedTask;
```

Comment:
I changed these to `await Task.CompletedTask` for consistency with how this is handled in the rest of the codebase